Add icon checklist workflow (#2071)
LawnchairLauncher/lawnicons: .github/icon_checklist.md

> Hey! Thanks for the contribution. While there is no reviewer, you can do a self-review. This will help to fix the reasons why the icons may not be suitable for Lawnicons. 

Reviewer: ```suggestion
Thanks for your contribution!

While waiting for a review from our team, you can do a self-review to ensure that your icons are suitable for Lawnicons.
```

> ### Canvas and sizes [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#canvas--sizes)

Reviewer: Using `→` isn't really accessible. You could replace it with `(more info)` instead or simply make the whole heading a link.

This goes to all of the headers using `→`.

Reviewer 2: They are specially made like this. It is necessary that they do not stand out too much relative to more relevant information for self-review. In terms of accessibility, it is enough that they are large and stand out in color.

Reviewer: I'll try to see any other alternative solution soon-ish, but sure, this will work for now.

Reviewer 2: I had the idea not to link to individual sections, but to link only to the guide. We can use it. For example, at the end, write "The Lawnicons guidelines".

Reviewer: Sure thing. Here's an example:

```markdown
### More information
For a complete guide on the icon guidelines, visit the [Lawnicons Contributing Guidelines](CONTRIBUTING.md).
```

Reviewer 2: I deleted it in the end, because the link to the full guidelines is in the PR description template.

1. Canvas: `192×192px`.

> 2. Non-square icons: the long side of the icons should be `160px`.

Reviewer: ```suggestion
2. Non-square icons: the length of the icons should be `160px`.
```

Reviewer 2: > the length of the icons should be `160px`.

The requirement is that the width or height should be 160px. In my understanding, "length" refers only to width.

3. Square icons: `154×154px`.

### Color, stroke width and rounding [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#foundation)
1. Color: black `#000`.
2. No fill. Base stroke width: `12px`. `14px`, `10px`, `8px` — depending on the shape of the icons. `6px` — for fine details.
3. Rounded ends and joins. 90° corners are rounded by `6-32px`.

### Naming [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#naming)

> 1. Names should match the official ones and contain no additional text.

Reviewer: ```suggestion
1. Names should match the official app name and contain no additional text.
```

2. If the original name is not in English, then add a localized (or transliterated) one to it using `~~`.
Example: `フールー ~~ Hulu`.
3. The names of the drawables should repeat the names of the apps if nothing prevents it.

### Quality [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#details)

> 1. Icons should be easily recognizable.

Reviewer: ```suggestion
1. Ensure that icons are easily recognizable.
```

2. Align icons to [the visual center](https://www.google.com/search?sca_esv=1fcec3f5e0b15e20&q=what+is+visual+center+of+an+icon&tbm=isch&source=lnms).

> 3. No blackness. Reduce the stroke width or simplify the icons so that there are no noticeable black spots.

Reviewer: ```suggestion
3. Avoid blackness or noticable black spots by reducing the stroke width or simplifying the icons.
```

4. Avoid close distances between strokes. The icons on the phone screen will be smaller, so the small distances between the strokes will stick together.

> 5. Avoid drastic changes in stroke widths. When the strokes next to each other differ in width by 4px or more, the icon will look sloppy.

Reviewer: ```suggestion
5. Avoid drastic changes in stroke widths. When the strokes next to each other differ in width by 4px or more, the icon will look sloppy.

```
I'd recommend adding a newline at the end of the file.
